Generated Cloudflare wrangler.toml does not work

A previous version of my `wrangler.toml` file was no longer working after upgrading React and waku (no functions were detected by wrangler), so I deleted it and regenerated the file. The new file generated by waku has the `main` and `assets` keys added, but does not work.

This is the file generated by waku (`0.21.8`) when running `waku build --with-cloudflare`:

```toml
# See https://developers.cloudflare.com/pages/functions/wrangler-configuration/
name = "waku-project"
compatibility_date = "2024-09-23"
compatibility_flags = [ "nodejs_als" ]
pages_build_output_dir = "./dist"
main = "./dist/worker/serve-cloudflare.js"

assets = {
  directory = "./dist/assets",
  binding = "ASSETS",
  html_handling = "drop-trailing-slash",
  # "single-page-application" | "404-page" | "none"
  not_found_handling = "404-page"
}
```

This is the error running `wrangler pages dev` (`3.94.0`):

```
✘ [ERROR] Unterminated inline array

    /.../wrangler.toml:9:9:
      9 │ assets = {
        ╵          ^
```

If I collapse the assets object to one line (`assets = { ... }`), I get this error instead:

```
✘ [ERROR] Processing wrangler.toml configuration:

    - The name 'ASSETS' is reserved in Pages projects. Please use a different name for your Assets
  binding.
```

--- a/packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts
+++ b/packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts
@@ -221,16 +221,14 @@ export function deployCloudflarePlugin(opts: {
 name = "waku-project"
 compatibility_date = "2024-09-23"
 compatibility_flags = [ "nodejs_als" ]
-pages_build_output_dir = "./dist"
 main = "./dist/worker/serve-cloudflare.js"
 
-assets = {
-  directory = "./dist/assets",
-  binding = "ASSETS",
-  html_handling = "drop-trailing-slash",
-  # "single-page-application" | "404-page" | "none"
-  not_found_handling = "404-page"
-}
+[assets]
+directory = "./dist/assets"
+binding = "ASSETS"
+html_handling = "drop-trailing-slash"
+# "single-page-application" | "404-page" | "none"
+not_found_handling = "404-page"
 `,
         );
       }

